docs/current_docs/manuals/developer/snippets/documentation/go/object.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+// The struct represents a single user of the system.
+type MyModule struct {
+	// The name of the user.
+	Name string
+	// The age of the user.
+	Age int
+}
+
+func New(name string, age int) *MyModule {
```

Comment:
The constructor's arguments aren't documented so these flags won't show as being documented.

Reply:
This didn't work. Am I doing something wrong?

```
package main

// The struct represents a single user of the system.
type MyModule struct {
	// The name of the user.
	Name string
	// The age of the user.
	Age int
}

func New(name string, age int) *MyModule {
	return &MyModule{
		// The name of the user.
		Name: name,
		// The age of the user.
		Age: age,
	}
}
```

```
FUNCTIONS
  age           The age of the user.
  name          The name of the user.

ARGUMENTS
      --age int       [required]
      --name string   [required]   
```

Reply:
This didn't work. Am I doing something wrong?

```
package main

// The struct represents a single user of the system.
type MyModule struct {
	// The name of the user.
	Name string
	// The age of the user.
	Age int
}

func New(name string, age int) *MyModule {
	return &MyModule{
		// The name of the user.
		Name: name,
		// The age of the user.
		Age: age,
	}
}
```

```
FUNCTIONS
  age           The age of the user.
  name          The name of the user.

ARGUMENTS
      --age int       [required]
      --name string   [required]
```

Reply:
Done